domains/eclipse.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "Just-a-Unity-Dev",
+        "email": "",
+        "twitter": "@eclips_e69"
+    },
+    "description": "A little portfolio based with a basic account system for learning purposes.",
+    "record": {
+        "CNAME": "eclips-espace-production.up.railway.app"
```

Comment:
application error?

Reply:
whoops, accidentally made railway shit itself. i'll try to fix it later with my mothers laptop

Reply:
oops